### DISCUSSION TITLE
安装出现错误提示

### DISCUSSION THREAD

**Discussion by SoultionLss** [MoeNetwork/Tieba-Cloud-Sign#257] [疑问]
php_network_getaddresses: getaddrinfo for db failed: Name does not resolve

**Comment by n0099** [comment]
https://github.com/MoeNetwork/Tieba-Cloud-Sign/blob/09ca411da65a97808898e639466f85808c491153/docker/docker-compose.yml#L18-L19

**Reply by SoultionLss** [reply]
是这个啊，没错啊，就是错了

**Reply by n0099** [reply]
https://docs.docker.com/engine/reference/commandline/network_inspect/
```bash
docker network inspect <network-name>
```

**Reply by SoultionLss** [reply]
不行，其他的没问题

**Reply by n0099** [reply]
别谜语

**Reply by SoultionLss** [reply]
就是docker其他项目可以正常数据库的，就这边不行

**Reply by n0099** [reply]
https://stackoverflow.com/questions/54433522/docker-compose-and-postgres-name-does-not-resolve
